tests/utils/call-cli.js

```diff
@@ -2,8 +2,15 @@ const execa = require('execa')
 
 const cliPath = require('./cli-path')
 
+const CLI_TIMEOUT = 3e5
+
 const callCli = async function (args, execOptions) {
-  const { stdout } = await execa(cliPath, args, { windowsHide: true, windowsVerbatimArguments: true, ...execOptions })
+  const { stdout } = await execa(cliPath, args, {
+    windowsHide: true,
```

Comment:
This is already [the default value](https://github.com/sindresorhus/execa#windowshide).
Note: `windowsHide` is a tricky option which has [pros and cons](https://github.com/sindresorhus/execa/issues/433). We probably want to keep the default value here and let Execa decides what the best value for it is.

Reply:
Removed in https://github.com/netlify/cli/pull/1727/commits/130ee481e20c00e3d4c45f9fd3ceea59414b28e7